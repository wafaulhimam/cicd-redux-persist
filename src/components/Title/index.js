import React from 'react';

const Title = () => {
    return (
        <p>Test 123</p>
    )
}

export default Title;