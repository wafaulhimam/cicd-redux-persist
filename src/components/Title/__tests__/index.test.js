import React from 'react';
import { shallow } from 'enzyme';
import Title from '../index';

const title = 'Test Title';
let wrapped = shallow(<Title>{title}</Title>)

describe('Component Title', () => {
    it('should render title component perfectly', () => {
        expect(wrapped).toMatchSnapshot();
    })
})

