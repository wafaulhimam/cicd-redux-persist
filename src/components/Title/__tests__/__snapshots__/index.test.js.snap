// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Title should render title component perfectly 1`] = `
<p>
  Test 123
</p>
`;

exports[`Title should render title component correctly 1`] = `ShallowWrapper {}`;
